src/assets/wise5/components/label/label-student/label-student.component.ts

```diff
@@ -346,9 +360,14 @@ export class LabelStudent extends ComponentStudent {
     this.canvas.getObjects('i-text').forEach((object: any) => {
       labels.push(this.getLabelJSONObjectFromText(object));
     });
+    labels.sort(this.sortByTimestamp);
     return labels;
   }
 
+  sortByTimestamp(labelA: any, labelB: any): number {
```

Comment:
Make function private and name more descriptive?
```suggestion
  private sortByTimestampOldestFirst(labelA: any, labelB: any): number {
```